src/node_report.cc

```diff
@@ -1030,14 +1039,20 @@ static void PrintResourceUsage(std::ostream& out) {
 #if defined(__APPLE__) || defined(_AIX)
     snprintf( buf, sizeof(buf), "%ld.%06d", stats.ru_utime.tv_sec, stats.ru_utime.tv_usec);
     out << "\n  User mode CPU: " << buf << " secs";
+    cpu_abs = std::stod(buf);
     snprintf( buf, sizeof(buf), "%ld.%06d", stats.ru_stime.tv_sec, stats.ru_stime.tv_usec);
     out << "\n  Kernel mode CPU: " << buf << " secs";
+    cpu_abs += std::stod(buf);
 #else
     snprintf( buf, sizeof(buf), "%ld.%06ld", stats.ru_utime.tv_sec, stats.ru_utime.tv_usec);
     out << "\n  User mode CPU: " << buf << " secs";
+    cpu_abs = std::stod(buf);
     snprintf( buf, sizeof(buf), "%ld.%06ld", stats.ru_stime.tv_sec, stats.ru_stime.tv_usec);
     out << "\n  Kernel mode CPU: " << buf << " secs";
+    cpu_abs += std::stod(buf);
 #endif
+    cpu_percentage = (cpu_abs / diff_time) * 100;
+    out << "\n  Average Consumption : "<< cpu_percentage << "%";
```

Comment:
What number do we actually want to show here? Does 100% mean all the processing time across all CPUs or just 100% of one CPU?
To put it another way, does the calculation for RUSAGE_SELF need to be divided by the number of availble CPUs to get the fraction of CPU time used in the whole system? (The value for RUSAGE_THREAD won't as a thread can by definition only use one CPU.)

As it stands the code could report that the whole process used 400% of CPU time when in fact on an 8 way box that might need to only be 50% of the total available. Which one does the user expect? What other tools will this number be compared against and what do they do? (e.g. what would `top` show?)